data/assets/mas.scene

```diff
@@ -0,0 +1,179 @@
+        "openspace.setPropertyValueSingle('Scene.FL_MAS.renderable.Flow.particleSpacing', 59);" ..
+        "openspace.setPropertyValueSingle('Scene.FL_MAS.renderable.Flow.speed', 19);" ..
+        "openspace.setPropertyValueSingle('Scene.FL_MAS.renderable.Color.uniform', {0.575, 0, 0.75, 1});" ..
+        "openspace.setPropertyValueSingle('Scene.FL_MAS.renderable.Flow.color', {1, 0, 0.65, 1});" ..
+        "openspace.setPropertyValueSingle('Scene.Sun.renderable.ColorTexture', MasScene.SunTextures[1]);"
+end
+
+local Keybindings = {
+    {
+        Key = "F5",
+        Command = "openspace.time.setTime('2000 JUL 14 08:33:37.100');",
+        Documentation = "Jumps to first of the MAS-fieldline timesteps",
+        Local = true
+    },
+    {
+        Key = "F6",
+        Command = "openspace.time.setTime('2000 JUL 14 10:11:50.100');",
+        Documentation = "Set time close to when bastille eruption starts",
+        Local = false
+    },
+    {
+        Key = "F7",
+        Command = "openspace.time.setTime('2012 JUL 01 00:07:00.000');",
+        Documentation = "Jumps to first of the Enlil-fieldline timesteps",
+        Local = true
+    },
+    {
+        Key = "F8",
+        Command = propertyHelper.invert('Scene.SunGlare.Renderable.Enabled'),
+        Documentation = "Toggle sun glare",
+        Local = false
+    },
+    {
+        Key = "F9",
+        Command = propertyHelper.invert('Scene.Sun.Renderable.Enabled'),
+        Documentation = "Toggle sun",
+        Local = false
+    },
+    {
+        Key = "F10",
+        Command = setup(),
+        Documentation = "Setup scene property values after initialization",
+        Local = false
+    },
+    {
+        Key = "i",
+        Command = [[
+            if (MasScene.Options.TexIndex == nil) then 
+                MasScene.Options.TexIndex = 1 
+            end; 
+
+            MasScene.Options.TexIndex = MasScene.Options.TexIndex + 1
+            if (MasScene.Options.TexIndex > #MasScene.SunTextures) then
+                MasScene.Options.TexIndex = 1 
+            end;
+
+            openspace.setPropertyValueSingle('Scene.Sun.renderable.ColorTexture', MasScene.SunTextures[MasScene.Options.TexIndex]);
+        ]],
+        Documentation = "Display next sun texture in list of textures",
+        Local = false
+    },
+    {
+        Key = "t",
+        Command = propertyHelper.invert('Scene.Mas_MHD_temperature.renderable.Enabled'),
+        Documentation = "Toggle mas temperature asset on/off",
+        Local = true
+    },
+    {
+        Key = "d",
+        Command = propertyHelper.invert('Scene.Mas_MHD_rho.renderable.Enabled'),
+        Documentation = "Toggle mas rho asset on/off",
+        Local = true
+    },
+}
+
+-- FIELDLINES --
+
+-- We need to specify paths to the prepared fieldline files before requesting the BATSRUS, ENLIL, and PFSS fieldline assets:
+-- local FieldLineBasePath = "E:/FieldlineBinaries/"
```

Comment:
this path stuff appears to be commented out and thus not needed?

Reply:
actually this scene is not being used. I'll remove it for now, but ill keep it locally because I'm unsure of what Scene.FL_MAS.renderable is.